README.md

```diff
@@ -180,7 +188,10 @@ Diff-Split: +250156  -0
 ```
 
 # How to publish to PyPI
```

Comment:
Why do we have instructions on publishing to PyPI? We are the only ones who will do it. (and probably 1 specific person who's in charge of it)

Reply:
I think we should remove it. It will confuse users, if nothing else.

It's fine if we create PUBLISH.md if we just want to be perfectly open, I just don't think it belong in the README.